src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx

```diff
@@ -165,18 +181,18 @@ const GeneSummary = () => {
 
       <div className={rowClasses}>
         <div className={styles.value}>
-          {`${gene.transcripts.length} transcripts`}
+          {gene.transcripts.length}{' '}
+          {pluralise('transcript', gene.transcripts.length)}
         </div>
       </div>
 
       <div className={classNames(rowClasses, styles.downloadRow)}>
         <div className={styles.value}>
-          <div
-            className={styles.downloadLink}
+          <ShowHide
+            label="Download"
+            isExpanded={shouldShowDownload}
             onClick={() => showDownload(!shouldShowDownload)}
```

Comment:
- No longer need the close button
- Should InstantDownload component appear to the right of the ShowHide, as it does now, or below it?

<img src="https://user-images.githubusercontent.com/6834224/134322064-38feb565-a914-44b2-84b9-5bdedc6acff8.png" width="400"/>

Reply:
As a reminder: also use ShowHide in TranscriptSummary component.